### PR TITLE
GCC 6.4.0 + binutils 2.28 (base for */2017b toolchains)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GCCcore/GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    'http://ftpmirror.gnu.org/gnu/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/gnu/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://isl.gforge.inria.fr/',  # original HTTP source for ISL
+]
+
+sources = [
+    'gcc-%(version)s-RC-20170628.tar.gz',  # FIXME 'gcc-%(version)s.tar.bz2',
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.4.0.eb]: currently testing with release candidate for GCC 6.4.0, this needs to be changed before merging

Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.4.0.eb]: final release is expected next week, see https://gcc.gnu.org/ml/gcc/2017-06/msg00131.html

**Comment by boegel** [comment]
Because the release candidate for GCC 6.4.0 uses `6.3.1` as version internally, the sanity check for GCCcore currently fails.
This should be fixed once the proper release is used instead.

**Comment by boegel** [comment]
@wpoely86 please review?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2075.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/990dc362393ea39c28db48353b6b76c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2699.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/81ea917a3c486ab48fba49f443ba0a85 for a full test report.

**Comment by boegel** [comment]
@easybuilders/easybuild-easyconfigs-maintainers good to go?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/2e8fb0a53901f95b3fac5a3526c92efe for a full test report.

**Comment by vanzod** [comment]
Thanks @boegel

**Pull request merged**